api/v1/jobs/PKPJobController.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * @file api/v1/jobs/PKPJobController.php
+ *
+ * Copyright (c) 2014-2023 Simon Fraser University
+ * Copyright (c) 2003-2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPJobController
+ *
+ * @ingroup api_v1_jobs
+ *
+ * @brief Handle API requests for jobs
+ *
+ */
+
+namespace PKP\API\v1\jobs;
+
+use APP\facades\Repo;
+use PKP\security\Role;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use PKP\core\PKPBaseController;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Route;
+
+class PKPJobController extends PKPBaseController
+{
+    public function isSiteWide(): bool
+    {
+        return true;
+    }
+
+    public function getHandlerPath(): string
+    {
+        return 'jobs';
+    }
+
+    public function getRouteGroupMiddleware(): array
+    {
+        $roles = implode('|', [Role::ROLE_ID_SITE_ADMIN]);
+
+        return [
+            "has.user",
+            "has.roles:{$roles}",
```

Comment:
I think it would be better to have these strings coming from a class, to avoid typos (e.g. `return [HasUserMiddleware::class, HasRolesMiddleware::for(Role::ROLE_ID_SITE_ADMIN)]` or something like that).